searching/linear_search.c

```diff
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+//function to receive parameters (array, size and value) and start searching linearly.
```

Comment:
For how to document functions, please see the [repository standards](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md).